### PR TITLE
add memgraph_kitty: display memgraphs in the kitty terminal emulator

### PR DESCRIPTION
**Pull request by Armael**

### CODE REVIEW

**Comment by Armael** [comment]
TODO (for tomorrow): write some documentation in `memgraph_kitty.mli`.

**Comment by Armael** [comment]
Done. This is ready for review!

File: memgraph_kitty.opam
Comment (Gbury) [inline memgraph_kitty.opam]: You could probably add yourself to the author and/or maintainer list I suppose (not sure how to do that with dune generating these files though).

File: src/kitty/memgraph_kitty.ml
Comment (Gbury) [inline src/kitty/memgraph_kitty.ml:3]: There should probably be some header added at the top of this file (even if only a one-line comment referring to the LICENSE file like in the other files).

File: dune-project
Comment (Gbury) [inline dune-project:35]: you should probably add a dependency on `conf-graphviz` (which already handles the adequate depexts to make sure `dot` is available).

**Review by Gbury — CHANGES_REQUESTED** [review]
Overall this looks good. A few remarks here and there, and once they are fixed, I'll merge this.

I wonder whether it would be possible to have a test of `memgraph_kitty`, but I guess that would be quite complicated to setup.

File: dune-project
Comment (Armael) [reply dune-project:35]: there is! (the line below)

File: src/kitty/memgraph_kitty.ml
Comment (Armael) [reply src/kitty/memgraph_kitty.ml:3]: done I think

File: memgraph_kitty.opam
Comment (Armael) [reply memgraph_kitty.opam]: done

File: dune-project
Comment (Gbury) [reply dune-project:35]: oups, I guess I glossed over that because it didn't have version constraint, my bad, ^^

**Review by Gbury — APPROVED** [review]
Thanks !

**Pull request merged**
